libvirt/resource_libvirt_domain.go

```diff
@@ -1215,6 +1220,38 @@ func newDiskForCloudInit(virConn *libvirt.Connect, volumeKey string) (libvirtxml
 	return disk, nil
 }
 
+func destroyDomainByUserRequest(d *schema.ResourceData, domain *libvirt.Domain) error {
+	// if the running is true in the tf file, we don't destroy the domain
+	if !destroyDomainRequestedByUser {
```

Comment:
As I said, couldn't we get the `destroyDomain` as an attribute? like `d.Get('destroyDomain')` or something like that?

Reply:
i'm thinking now how to solve this, i will address this asap